packages/backend/src/plugins/search.ts

```diff
@@ -73,7 +73,14 @@ export default async function createPlugin({
 
   indexBuilder.addCollator({
     defaultRefreshIntervalSeconds: 600,
-    collator: new DefaultTechDocsCollator({ discovery, logger }),
+    collator: DefaultTechDocsCollator.fromConfig(config, {
+      discovery,
+      logger,
+      legacyPathCasing:
+        config.getOptionalBoolean(
+          'techdocs.legacyUseCaseSensitiveTripletPaths',
```

Comment:
Does this mean that we now have a config parameter in place, that may not actually be read by most installations so it can lead to confusion when people ask for support? Why is not this one read inside `fromConfig`? I don't get that bit.

Reply:
Good point. I modified this to be read directly within the collator instead.